src/main/java/it/gov/pagopa/pu/migration/service/file/ErrorArchiverService.java

```diff
@@ -0,0 +1,117 @@
+package it.gov.pagopa.pu.migration.service.file;
```

Comment:
this is a service dedicated to the Activities, let's change its package accordingly
```suggestion
package it.gov.pagopa.pu.migration.wf.service.ingestion;
```